main/src/main/java/org/qbicc/main/Main.java

```diff
@@ -229,6 +230,8 @@ DiagnosticContext call0(BaseDiagnosticContext initialContext) {
                             // for now, type IDs == int32
                             tsBuilder.setTypeIdSize((int) probeResult.getTypeInfo(int32_t).getSize());
                             tsBuilder.setTypeIdAlignment((int) probeResult.getTypeInfo(int32_t).getAlign());
+                            // todo: endianness probe
+                            tsBuilder.setEndianness(ByteOrder.LITTLE_ENDIAN);
```

Comment:
You should be able to get this value from `probeResult.getByteOrder()` already.

Reply:
done